lib/ethernet/subsystem.ex

```diff
@@ -113,7 +113,7 @@ defmodule Nerves.IO.Ethernet.Subsystem do
   end
 
   defp ip_cmd(cmd) do
-    "/sbin/ip " <> cmd
+    "/sbin/ip #{cmd}"
```

Comment:
@ghitchens This change was just to begin the *data transformation* with a bare value instead of a concatenation.